CMakeLists.txt

```diff
@@ -18,4 +18,5 @@ cmake_minimum_required(VERSION 3.6 FATAL_ERROR)
 project(SplineLib VERSION 0.0.1 LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
 
-add_subdirectory(test)
\ No newline at end of file
+add_subdirectory(test)
```

Comment:
Please change the order of test and src. test depends on src. Therefore, src should be in front of test.